packages/svelte/src/lib/index.ts

```diff
@@ -31,6 +31,7 @@ export * from '$lib/hooks/useConnection';
 export * from '$lib/hooks/useNodesEdges';
 export * from '$lib/hooks/useHandleConnections';
 export * from '$lib/hooks/useNodesData';
+export * from '$lib/hooks/useInitialized';
```

Comment:
we need to export `useNodesInitialized` / `useEdgesInitialized` (do we want to offer that?) too

Reply:
Oh wait `useInitialized` includes `useNodesInitialized` `useEdgesInitialized`. It is already exported